### PR TITLE
Use css columns

### PR DESCRIPTION
**Pull request by nwhite89**
Allows setting how many columns to display a control groups contents in.

TP: https://rockabox.tpondemand.com/entity/9817

### CODE REVIEW

File: test/unit/rb-check-control/rb-check-control-group.spec.js [hunk 1]
```diff
@@ -65,6 +65,30 @@ define([
...
+
+                compileTemplate('<rb-check-control-group options="options" ng-model="model" columns="2">' +
+                    '</rb-check-control-group>');
+
+                var groupBody = angular.element(element.find('div')[1]);
+
+                expect(groupBody.hasClass('u-columnCount2')).toBeTruthy();
+            });
+
+            it('should set the group to display in columns', function () {
```
Comment (chrisvxd) [inline test/unit/rb-check-control/rb-check-control-group.spec.js]: Can you ensure we have unique test case names?

**Comment by chrisvxd** [comment]
Good besides changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 
 - [`rb-page-title` supports transcluding any action component by convention, not just buttons](https://github.com/rockabox/rbx_ui_components/pull/217)
+- [`rb-check-control-group` supports setting content in groups](https://github.com/rockabox/rbx_ui_components/pull/238)
```
Comment (chrisvxd) [inline CHANGELOG.md]: Should be alphabetical but will ignore since it is already out of order

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
+- [`rb-check-control-group` supports setting content in groups](https://github.com/rockabox/rbx_ui_components/pull/238)
```
Comment (chrisvxd) [inline CHANGELOG.md]: Don't see how this captures the ticket? Should mention columns, something like:

"[`rb-check-control-group` `columns` attribute](https://github.com/rockabox/rbx_ui_components/pull/238) Split group into columns using css"

Comment (nwhite89) [inline CHANGELOG.md]: ok will ammend

**Pull request merged**